core/src/main/java/org/apache/carbondata/core/metadata/encoder/Encoding.java

```diff
@@ -57,10 +62,55 @@ public static Encoding valueOf(int ordinal) {
       return ADAPTIVE_DELTA_INTEGRAL;
     } else if (ordinal == RLE_INTEGRAL.ordinal()) {
       return RLE_INTEGRAL;
+    } else if (ordinal == DIRECT_STRING.ordinal()) {
+      return DIRECT_STRING;
+    } else if (ordinal == ADAPTIVE_FLOATING.ordinal()) {
+      return ADAPTIVE_FLOATING;
+    } else if (ordinal == BOOL_BYTE.ordinal()) {
+      return BOOL_BYTE;
+    } else if (ordinal == ADAPTIVE_DELTA_FLOATING.ordinal()) {
+      return ADAPTIVE_DELTA_FLOATING;
     } else if (ordinal == DIRECT_COMPRESS_VARCHAR.ordinal()) {
       return DIRECT_COMPRESS_VARCHAR;
     } else {
       throw new RuntimeException("create Encoding with invalid ordinal: " + ordinal);
     }
   }
+
+  /**
+   * Method to validate for supported encoding types that can be read using the current version
+   *
+   * @param encodings
+   * @return
+   */
+  public static boolean assertTrueForEncodingTypes(
```

Comment:
1)This method return value is never used. So please remove the return type.
2) Change method name validateEncodingTypes

Reply:
done